assembl/models/langstrings_helpers.py

```diff
@@ -43,9 +43,9 @@ def langstrings_attrs_dict(langstrings_names):
     return d
 
 
-def LangstringsBase(langstrings_names):
+def LangstringsBase(langstrings_names, model_name):
```

Comment:
As this is a method, can you please make it pythonic snake_case? `langstring_base`.

Reply:
The input here should have some sanitization, at min, a `string.trim`. Perhaps comment the `:param model_name` so that the format is clear for future usage.